.travis.yml

```
@@ -1,23 +1,13 @@
 language: ruby
 sudo: false
 
-rvm:
-  - 2.2.5
-  - 2.3.1
-
-gemfile:
-  - Gemfile.rails42
-  - Gemfile.rails50
-  - Gemfile.rails51
-  - Gemfile.rails52
-
-matrix:
+jobs:
   include:
-    - rvm: 2.1.9
-      gemfile: Gemfile.rails42
-    - rvm: 2.5.0
+    - rvm: 2.4.9
```

We should also be able to omit the bugfix release and get the latest bugfix release.  E.g.

```suggestion
    - rvm: 2.4
```

---

Ah, right. Done!

---

Actually, it seems to default to `2.4.6`. 🤔

---

Oh well.  I don't think it matters enough to workaround unless it actually breaks the tests.